packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js

```diff
@@ -228,6 +263,16 @@ QUnit.module('options', {
         assert.equal(this.tabWidgetInstance.option('selectedItem'), this.items[1], 'option <selectedItem> of nested tabs widget successfully changed');
     });
 
+    QUnit.test('orientation tabs option test', function(assert) {
+        assert.expect(2);
```

Comment:
I don't think it makes any sense

Reply:
I agree.

Reply:
[feccc1664117de61365ba2df824d600fa6ed06f6]